src/functionalTest/resources/scenarios/RIA-4564-RIA-2917-respondent-notification-of-change-direction-due-date-awaitingReasonsForAppeal-aip.json

```diff
@@ -0,0 +1,122 @@
+              "wantsEmail": "Yes",
+              "mobileNumber": "{$TEST_CITIZEN_MOBILE}",
+              "wantsSms": "Yes"
+            }
+          }
+        ],
+        "directionEditExplanation": "Upload additional evidence",
+        "directionEditDateDue": "{$TODAY+14}",
+        "directionEditParties": "respondent",
+        "notificationsSent": [
+          {
+            "id": "45641_RESPONDENT_CHANGE_DIRECTION_DUE_DATE",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          },
+          {
+            "id": "45641_APPELLANT_CHANGE_DIRECTION_DUE_DATE_OF_HOME_OFFICE_EMAIL",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          },
+          {
+            "id": "45641_APPELLANT_CHANGE_DIRECTION_DUE_DATE_OF_HOME_OFFICE_SMS",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          }
+        ],
+        "listCaseHearingCentre": "birmingham",
+        "currentCaseStateVisibleToHomeOfficeAll": "awaitingReasonsForAppeal"
+      }
+    },
+    "notifications": [
+      {
+        "reference": "45641_RESPONDENT_CHANGE_DIRECTION_DUE_DATE",
+        "recipient": "{$apcHomeOfficeEmailAddress}",
+        "subject": "Immigration and Asylum appeal: change to direction due date",
+        "body": [
+          "PA/12345/2019",
+          "A1234567",
+          "Pablo Jimenez",
+          "{$iaExUiFrontendUrl}",
+          "{$TODAY+14|d MMM yyyy}",
+          "{$customerServices.telephoneNumber}",
+          "{$customerServices.emailAddress}"
+        ]
+      },
+      {
+        "reference": "45641_APPELLANT_CHANGE_DIRECTION_DUE_DATE_OF_HOME_OFFICE_EMAIL",
+        "recipient": "{$TEST_CITIZEN_USERNAME}",
+        "subject": "Immigration and Asylum appeal: The Tribunal has changed a date for the Home Office to do something",
+        "body": [
+          "PA/12345/2019",
+          "A1234567",
+          "Pablo Jimenez",
+          "The Tribunal recently asked the Home Office to do something as part of your appeal. The Tribunal has now changed the date it gave the Home Office to do this by.",
+          "This is the message the Tribunal sent the Home Office that includes the new date:",
+          "Upload additional evidence",
+          "{$TODAY+14|d MMM yyyy}",
+          "You don’t need to do anything. If you have any questions, you should contact the Tribunal.",
+          "{$customerServices.telephoneNumber}",
+          "{$customerServices.emailAddress}"
+        ]
+      },
+      {
+        "reference": "45641_APPELLANT_CHANGE_DIRECTION_DUE_DATE_OF_HOME_OFFICE_SMS",
+        "recipient": "{$TEST_CITIZEN_MOBILE}",
+        "subject": "",
+        "body": [
+          "PA/12345/2019",
+          "The Tribunal recently asked the Home Office to do something as part of your appeal. The Tribunal has now changed the date it gave the Home Office to do this by.",
```

Comment:
Hi @jkumar-moj. Just a question on testing the static content inside the templates. We previously moved away from this (and only tested the dynamic/placeholder values inside the template) because when the content designers changed the static content inside the GovNotify templates, the functional tests would fail.

Reply:
Thanks @bradsorour I noticed it in other places but I was not aware of that. I deliberately added it to make sure to avoid template changes. I will remove the content check and update the PR